### ISSUE TITLE
Editor Component: a11y problems

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/codemirror-blocks#165]
- [ ] Trashcan should be visible at all times (especially true for users on the spectrum)
- [x] When toolbar is not visible, it needs `aria-hidden=true` and `tabIndex=-1`
- [x] Focusing across regions is badly broken
- [x] `aria-label`s needed _errrrverywhere_.
- [x] `aria-role=treeitem`s need to be stripped from all the primitives, since this is _not_ a tree view

**Comment by schanzer** [comment]
The codebase has changed so significantly since this issue was opened that we should close it - esp since the four solved items _remain_ solved in the new codebase, and we have another issue for the unsolved one (#215)
